packages/charts/src/charts/table/components/selection-column/header/header.component.tsx

```diff
@@ -18,12 +18,20 @@ export const Header = ({
 
   return (
     <HeaderCellContent onClick={() => !editMode && toggleAllRowsSelected()}>
-      <Checkbox
-        id="select-all-rows"
-        type="highlight"
-        display="inline-flex"
-        checked={checked}
-      />
+      <MousePositionedTooltip
+        isActive
+        tooltipPinPlacement="bottom-right"
+        renderContent={() =>
+          checked ? 'Unselect all rows' : 'Select all rows'
```

Comment:
just wondering if we want translate these strings later on

Reply:
Yep, we can add translation props in next iteration.